### PR TITLE
feat(issue-28): write a test code and split the code

### PR DESCRIPTION
**Pull request by jskurt**
## Issue Link

[issue-28](https://github.com/kafelix496/coding-for-fun/issues/28)

## Notes

-

### CODE REVIEW

File: src/modules/github/components/pull-review-form/utils.test.ts [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+            user: { login: 'user2' }
+          }
+        ]
+      }
+    ]
+
+    const result = getCheckedPullsInfo(repoHasCheckArray)
+
+    expect(result).toEqual([])
+  })
```
Comment (kafelix496) [inline src/modules/github/components/pull-review-form/utils.test.ts:66]: it('should return empty array if there're no checked pull', ...)

File: src/modules/github/components/pull-review-form/utils.test.ts [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+      }
+    ]
+
+    const result = getCheckedPullsInfo(repoHasCheckArray)
+
+    expect(result).toEqual([
+      { org: 'org1', repo: 'repo1', pullTitle: 'pull1-1', pullNumber: 1 - 1 },
+      { org: 'org1', repo: 'repo2', pullTitle: 'pull2-1', pullNumber: 2 - 1 }
+    ])
+  })
```
Comment (kafelix496) [inline src/modules/github/components/pull-review-form/utils.test.ts:113]: 1. for the `pullNumber`, it seems so weird. just put the regular number.
2. it('should return checked pulls information', ...)

File: src/modules/github/components/pull-review-form/utils.test.ts [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+        ]
+      }
+    ]
+
+    const result = getCheckedPullsInfo(repoHasCheckArray)
+
+    expect(result).toEqual([
+      { org: 'org1', repo: 'repo1', pullTitle: 'pull1', pullNumber: 1 },
+      { org: 'org1', repo: 'repo1', pullTitle: 'pull3', pullNumber: 3 }
+    ])
```
Comment (kafelix496) [inline src/modules/github/components/pull-review-form/utils.test.ts:39]: put this thing inside `it('should return checked pulls information', ...)`

**Comment by jskurt** [comment]
I finished resolved conflict from rebase !.

**Review by kafelix496 — APPROVED** [review]

**Pull request merged**